sw/device/tests/meson.build

```diff
@@ -225,11 +225,20 @@ foreach sw_test_name, sw_test_info : sw_tests
   endforeach
 endforeach
 
+# Signing keys for ROM_EXT images.
+signing_keys = {
+  'fpga_key_0': {
+    'path': meson.source_root() / 'sw/device/silicon_creator/keys/fpga_key_0.private.der',
+  },
+  'fpga_key_1': {
+    'path': meson.source_root() / 'sw/device/silicon_creator/keys/fpga_key_1.private.der',
```

Comment:
Since these keys are used on Verilator now as well, how about renaming them to dev keys or something along those lines? (can be done in a follow-up)

Reply:
These were initially named `dev_key_*` but @cfrantz pointed out that using a name like `dev_key_*` or `test_key_*` may suggest that these keys were endorsed to be used as such in silicon. Since they are not, we went with `fpga_key_*`.